subsys/mgmt/mcumgr/lib/cmd/img_mgmt/src/img_mgmt.c

```diff
@@ -363,6 +367,10 @@ img_mgmt_upload(struct mgmt_ctxt *ctxt)
 	int rc;
 	struct img_mgmt_upload_action action;
 	bool last = false;
+#ifdef CONFIG_IMG_ENABLE_IMAGE_CHECK
```

Comment:
Bring this down to `if (req.off == 0) {` block, they are only used there.

Reply:
Fixed